commit 230adc7929685b8aa9a37bc476f36bf5f4dd5b37
Author: Jaroslav Kysela <perex@perex.cz>
Date:   Wed Feb 15 17:09:42 2017 +0100

    ALSA: hda - fix Lewisburg audio issue
    
    commit e7480b34ad1ab84a63540b2c884cb92c0764ab74 upstream.
    
    Like for Sunrise Point, the total stream number of Lewisburg's
    input and output stream exceeds 15 (GCAP is 0x9701), which will
    cause some streams do not work because of the overflow on
    SDxCTL.STRM field if using the legacy stream tag allocation method.
    
    Fixes: 5cf92c8b3dc5 ("ALSA: hda - Add Intel Lewisburg device IDs Audio")
    Signed-off-by: Jaroslav Kysela <perex@perex.cz>
    Signed-off-by: Takashi Iwai <tiwai@suse.de>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1233_230adc7929685b8aa9a37bc476f36bf5f4dd5b37.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1233_230adc7929685b8aa9a37bc476f36bf5f4dd5b37.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
